packages/expo-updates/android/src/main/java/expo/modules/updates/logging/UpdatesLogReader.kt

```diff
@@ -0,0 +1,39 @@
+package expo.modules.updates.logging
+
+import java.io.IOException
+import java.util.*
+
+/**
+ * Class for reading expo-updates logs
+ */
+class UpdatesLogReader {
+
+  /**
+   Get expo-updates logs newer than the given date
+   Returns a list of strings in the JSON format of UpdatesLogEntry
+   */
+  fun getLogEntries(newerThan: Date): List<String> {
+    val epochTimestamp = newerThan.time / 1000
+    val pid = "${android.os.Process.myPid()}"
+    try {
```

Comment:
Awesome! Last thing on this one is that we probably want to not catch the error (just like in iOS).

Reply:
Done